packages/amplication-client/src/Workspaces/hooks/useResources.ts

```diff
@@ -28,7 +32,15 @@ const useResources = (currentProject: models.Project | undefined) => {
 
   useEffect(() => {
     if (loadingResources || !resourcesData) return;
-    setResources(resourcesData.resources);
+    const projectConfigurationResource = resourcesData.resources.find(
```

Comment:
you can combine those two loops to one reduce

Reply:
The list of projects will often include a few items and not more then a few dozens in most extreme cases. 
Also, the find "loop" will stop after finding the first item. So, for readability, I think it is better to keep the current code instead of quizzing another micro improvement